docs/Transformers.md

```diff
@@ -0,0 +1,108 @@
+# Transformers
+
+Transformers have three roles: to transform an assets content from one form to another, to create additional assets, and to register dependencies and connected files.
+
+### Transform / Compile
+
+Covertes the assets content from one structure to another.
+
+**Common Examples**
+
+- Babel: Transforms latest JavaScript to browser compatible JavaScript
+- SASS: Compiles sass files into css files
+
+### Create Assets
+
+Dynamic create new assets based on the assests provided.
+
+**Common Examples**
+
+- Babel: Transforms latest JavaScript to browser compatible JavaScript
+- SASS: Compiles sass files into css files
+- CSSModules: Produces JavaScript files to be imported
```

Comment:
I'm not entirely sure how this is meaningfully different than "Transform / Compile"

Reply:
I agree this is an example the cross over into both. I put it here because it produced a new file in addition to transpiling. 

I am open to changed on this. Just went with that I was aware of.